tests/queries/0_stateless/01548_query_log_query_execution_ms.sql

```diff
@@ -12,7 +12,7 @@ from system.query_log
 where
     query like '%01548_query_log_query_execution_ms%'
     and current_database = currentDatabase()
-    and query_duration_ms between 100 and 800
+    and query_duration_ms between 400 and 3000
```

Comment:
Why do you need an upper limit on the query duration? Maybe lower limit of 0.4 seconds is enough?

Reply:
I decided to leave the upper limit to ensure that 0.4 will not be 4 seconds (by some strange reason), but I'm fine with removing it altogether